corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -938,7 +938,8 @@ hqDefine("cloudcare/js/form_entry/entries", [
     FileEntry.prototype = Object.create(EntrySingleAnswer.prototype);
     FileEntry.prototype.constructor = EntrySingleAnswer;
     FileEntry.prototype.onPreProcess = function (newValue) {
-        var self = this;
+        const self = this;
+        const cachedFilename = self.question.form().fileNameCache[self.answer()];
```

Comment:
what's the distinction between `self.answer()` and `self.question.answer()`?

Reply:
There is none. self.answer just references to self. question.answer